maat-court-data-api/src/test/java/gov/uk/courtdata/hardship/impl/HardshipReviewImplTest.java

```diff
@@ -61,6 +63,23 @@ void givenExistingRepId_whenFindByRepIdIsInvoked_thenHardshipIsRetrieved() {
         assertThat(returnedEntity.getRepId()).isEqualTo(MOCK_REP_ID);
     }
 
+    @Test
+    void givenExistingRepId_whenFindByDetailTypeIsInvoked_thenHardshipIsRetrieved() {
+        when(hardshipReviewRepository.findByDetailType(MOCK_DETAIL_TYPE, MOCK_REP_ID))
+                .thenReturn(Optional.of(
+                        HardshipReviewEntity.builder()
+                                .id(MOCK_HARDSHIP_ID)
+                                .repId(MOCK_REP_ID)
+                                .build()));
+
+        Optional<HardshipReviewEntity> returnedEntity = hardshipReviewImpl.findByDetailType(MOCK_DETAIL_TYPE, MOCK_REP_ID);
+
+        if (returnedEntity.isPresent()) {
```

Comment:
I think the `if` condition should be removed, as if no entity is returned (not present) the test should fail, rather than silently succeed and skip the validation.

`if (returnedEntity.isPresent()) `